packages/scratch-vm/src/extensions/scratch3_spotify/index.ts

```diff
@@ -0,0 +1,492 @@
+import { ArgumentType, BlockType } from "../../typescript-support/enums";
+import { Extension } from "../../typescript-support/Extension";
+import {
+    Block,
+    DefineBlock,
+    Environment,
+    ExtensionMenuDisplayDetails,
+} from "../../typescript-support/types";
+import defineTranslations from "./translations";
+import Tone from "./tone.js";
+
+// FEEDBACK is it ok to still use require like this?
+const nets = require("nets");
```

Comment:
Ya, this should still work! But I think the 'better' way (better meaning I think it will give us proper type info) is to do:

```ts
import nets = require("nets");
```

But that's another thing I can double check!

Reply:
Hm, I think this works, but `nets` is old enough (and not maintained) so that there's no types for it. 

I think it actually be _good_ to not use `nets` anymore since the `fetch` API + javascript's async/await has come along way, so that they kind of code ends up looking a lot nicer/more readable.

Reply:
Ah, thanks for the tip about fetch. I'll check it out and use it in the future